qutebrowser/misc/sessions.py

```diff
@@ -459,6 +464,34 @@ def list_sessions(self):
                 sessions.append(base)
         return sorted(sessions)
 
+    def add_url_to_session(self, name):
+        """Appends the current tab's URL to a named session."""
+        data = self._load_session_file(name)
+
+        if data is None:
+            raise SessionError("Got empty session file")
+
+        winlist = objreg.window_registry
+        current_tab = None
+        for win_id in sorted(winlist):
+            tabbed_browser = objreg.get('tabbed-browser', scope='window',
+                                        window=win_id)
```

Comment:
I think you can pass `window='current'` in order to reference the current window (which should allow you to remove this loop)